Source/JavaScriptCore/wasm/WasmAirIRGeneratorBase.h

```diff
@@ -806,6 +806,19 @@ struct AirIRGeneratorBase {
             return m_proc.addConstant(B3::Origin(), toB3Type(tmp.type()), 0);
         };
 
+#if !USE(BUILTIN_FRAME_ADDRESS) || ASSERT_ENABLED
+        // Prepare wasm operation calls.
+        {
+            auto* patchpoint = addPatchpoint(B3::Void);
+            patchpoint->effects = B3::Effects::forCall();
+            patchpoint->setGenerator([=] (CCallHelpers& jit, const B3::StackmapGenerationParams& params) {
```

Comment:
I think this patchpoint can just be done in normal Air IR.

Reply:
Changed